clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -494,6 +501,14 @@ public void testSubscriptionOnEmptyPattern(GroupProtocol groupProtocol) {
             () -> consumer.subscribe(Pattern.compile("")));
     }
 
+    @ParameterizedTest
+    @EnumSource(value = GroupProtocol.class, names = "CONSUMER")
```

Comment:
You also need tests for the classic group protocol.

Reply:
Got it.

Reply:
nvm, please ignore my previous comment.